pkg/controllers/provisioning/provisioner.go

```diff
@@ -131,6 +131,14 @@ func (p *Provisioner) Reconcile(ctx context.Context, _ reconcile.Request) (resul
 	if len(results.NewNodeClaims) == 0 {
 		return reconcile.Result{}, nil
 	}
+	np , err := nodepoolutil.Get(ctx, p.kubeClient, results.NewNodeClaims[0].OwnerKey)
```

Comment:
I think this change may need a rebase since the `OwnerKey` field shouldn't exist anymore with some recent changes